.changeset/cool-grapes-attend.md

```diff
@@ -3,4 +3,5 @@
 'firebase': patch
 ---
 
-Fix Firestore failing to raise initial snapshot from empty local cache result
+Fix Firestore failing to raise initial snapshot from empty local cache result.
```

Comment:
All you have to do is put the issue number in brackets after the text with a leading hashtag.

See https://github.com/firebase/firebase-js-sdk/pull/6550 for an example. When published it will turn into the link text `Github issue #5873` (https://firebase.google.com/support/release-notes/js#version_994_-_september_1_2022).